docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "Installing Knative with Knative Operators"
+weight: 10
+type: "docs"
+---
+
+# Knative installations with Knative Operators
+
+Knative provides operators as tools to install, configure and manage Knative. This guide explains how to install and
+uninstall Knative using Knative operators.
+
+Each component in Knative has a separate operator for installation and configuration. This means that there is a [Serving operator](https://github.com/knative/serving-operator)
+and an [Eventing operator](https://github.com/knative/eventing-operator), and you can choose to install one or both independently.
+
+## Prerequisites:
+
+### Set up a Kubernetes cluster
+
+Knative operators are still in Alpha phase. They focus on installation on a generic Kubernetes platform, such as
+Docker Desktop, Minikube, or kubeadm clusters, and the operators do not perform any platform specific customization.
+If you are not sure the customization of the Knative operators for vendor-specific platforms, use the generic Kubernetes
+platform.
```

Comment:
I'd remove this section entirely since the only installation docs we offer now are the 'generic' ones: https://knative.dev/docs/install/any-kubernetes-cluster/

Maybe you could just re-use the **Before you begin** section of those docs here instead? cc @mattmoor to check the accuracy of this suggestion.

Reply:
@houshengbo idk if you missed this comment or you're just not doing this?
I don't think there's any need to mention specific vendors here. I'd remove lines 28-31 but it's your call.